python/lsst/sims/catUtils/supernovae/snObject.py

```diff
@@ -589,6 +590,94 @@ def SNObjectSED(self, time, wavelen=None, bandpass=None,
         SEDfromSNcosmo.addCCMDust(a_x=ax, b_x=bx, ebv=self.ebvofMW)
         return SEDfromSNcosmo
 
+    def SNObjectSourceSED(self, time, wavelen=None):
```

Comment:
If `source` is widely understood to mean "rest frame", then that is fine.  I don't think we should have a separate method that just aliases another.  Go ahead and remove `restFramSED()`.  Sorry for the back-and-forth.